apis/basic/composition.yaml

```diff
@@ -95,6 +100,8 @@ spec:
           patchSetName: network-id
         - type: PatchSet
           patchSetName: region
+        - type: PatchSet
+          patchSetName: network-id-tags
```

Comment:
nit: i would regroup and place this PatchSet next to network-id

Reply:
done